packages/common/src/store/pages/collection/actions.ts

```diff
@@ -7,11 +7,49 @@ export const FETCH_COLLECTION_FAILED = 'FETCH_COLLECTION_FAILED'
 export const RESET_COLLECTION = 'RESET_COLLECTION'
 export const SET_SMART_COLLECTION = 'SET_SMART_COLLECTION'
 
+export type FetchCollectionAction = {
+  type: typeof FETCH_COLLECTION
+  id: Nullable<number>
+  permalink?: string
+  fetchLineup?: boolean
+}
+
+export type FetchCollectionSucceededAction = {
+  type: typeof FETCH_COLLECTION_SUCCEEDED
+  collectionId: ID
+  collectionUid: string
+  collectionPermalink: string
+  userUid: string
+}
+
+export type FetchCollectionFailedAction = {
+  type: typeof FETCH_COLLECTION_FAILED
+  userUid: UID
+}
+
+export type ResetCollectionAction = {
+  type: typeof RESET_COLLECTION
+  collectionUid?: Nullable<UID>
```

Comment:
can we norm on `Nullable` vs `| null` since we're both doing so many type updates?

Reply:
also feels like maybe these don't need to be optional and nullable, but maybe it was already like that

Reply:
i like `| null` more personally